src/composables/useSortParticipants.js

```diff
@@ -99,6 +99,14 @@ export function useSortParticipants() {
 			return hasSessions1 ? -1 : 1
 		}
 
+		const p1IsPhone = participant1.actorType === ATTENDEE.ACTOR_TYPE.PHONES
+		const p2IsPhone = participant2.actorType === ATTENDEE.ACTOR_TYPE.PHONES
+
+		if (p1IsPhone !== p2IsPhone) {
```

Comment:
Need to check  position of phones in the list in group call against:
- participants in call
- online participants
- moderators